test-resources/swift/variable_inline/field_variable_inline/input/FieldVariableInline.swift

```diff
@@ -0,0 +1,507 @@
+// Copyright (c) 2023 Uber Technologies, Inc.
+
+// <p>Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file
+// except in compliance with the License. You may obtain a copy of the License at
+// <p>http://www.apache.org/licenses/LICENSE-2.0
+
+// <p>Unless required by applicable law or agreed to in writing, software distributed under the
+// License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+// express or implied. See the License for the specific language governing permissions and
+// limitations under the License.
+
+class C1{
+
+    var a = placeholder_true
+
+
+
+
+    init(){}
+
+
+
+
+    func f1(){
+
+        a = placeholder_true
+
+    }
+
+}
+
+
+
+
+class C2{
+
+    var a = placeholder_true
+
+
+
+
+    init(){}
+
+
+
+
+    func f2(){
+
+        a = placeholder_false
+
+    }
+
+}
+
+
+
+
+class C3{
+
+    var a = placeholder_true
+
+
+
+
+    init(){}
+
+
```

Comment:
can we please remove this unnecessary white space? Can u run a swift formatter over the swift files?

Reply:
I can do it for the final PR, will fix all these altogther.